docs/book/component-gallery/orchestrators/kubeflow.md

```diff
@@ -130,24 +130,40 @@ To use the Kubeflow orchestrator, we need:
 When using the Kubeflow orchestrator locally, you'll additionally need:
 * [K3D](https://k3d.io/v5.2.1/#installation) installed to spin up a local 
 Kubernetes cluster.
-* A [local container registry](../container-registries/default.md) as part of 
-your stack.
+* [Terraform](https://www.terraform.io/downloads.html) installed to set up the
+Kubernetes cluster with various deployments.
 
-{% hint style="warning" %}
-The local Kubeflow Pipelines deployment requires more than 2 GB of RAM,
-so if you're using Docker Desktop make sure to update the resource
-limits in the preferences.
-{% endhint %}
+To allow users to try out the Kubeflow orchestrator locally, we've created a
+terrafform recipe that sets up a local Kubernetes cluster with Kubeflow Pipelines
+and Minio on your machine. 
 
-We can then register the orchestrator and use it in our active stack:
+To deploy the stack, run the following commands:
 ```shell
-zenml orchestrator register <ORCHESTRATOR_NAME> \
-    --flavor=kubeflow
+# Pull the `k3d-modular` recipe to your local system
+zenml stack recipe pull k3d-modular
+# Deploy the stack using the ZenML CLI:
+zenml stack recipe deploy k3d-modular
+# run the following command to import the resources as a ZenML stack, manually
+zenml stack import <STACK_NAME> -f <PATH_TO_THE_CREATED_STACK_CONFIG_YAML>
+# set the imported stack as the active stack
+zenml stack set <STACK_NAME>
+```
 
-# Register and activate a stack with the new orchestrator
-zenml stack register <STACK_NAME> -o <ORCHESTRATOR_NAME> ... --set
+```shell
+# Get the Kubeflow Pipelines UI endpoint
+kubectl get ingress -n kubeflow  -o jsonpath='{.items[0].spec.rules[0].host}'
 ```
```

Comment:
I intended to separate it because in the recipe it gets all the ingress which include `mlflow` and `minio` one that are kinda persisted now in every recipe. But now we just want the UI of `kubeflow` itself.